.github/workflows/tests_and_formatting.yml

```diff
@@ -74,3 +74,16 @@ jobs:
       - name: Check if test succeeded
         if: steps.test.outcome != 'success'
         run: exit 1
+  prettier:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          ref: ${{ github.head_ref }}
+          fetch-depth: 0
+      - name: Prettify code
+        uses: creyD/prettier_action@v4.3
+        with:
+          prettier_options: --write --tab-width 4 /home/runner/work/fastn/fastn/fastn-js/js/**/*.js
+          only_changed: True
```

Comment:
Delete `prettier` "job" and put this `step` in `everything` job, else this will start one more VM, clone the repo etc just to run one extra `step`.